src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Smart Bulletin No. 036"
+intro: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies” Act or “CHARGE” Act)."
+number: "No. 036"
+order: 036
+businessline: "Purchase, Travel, Fleet and Integrated"
+published: "Mar 10, 2023"
+lastupdated: "Feb 13, 2023"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| March 10, 2023 | Implementation of Smart Bulletin #36 |
+| February 13, 2023 | Removed broken link |
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+This Smart Bulletin provides operational guidance regarding Public Law 116-160, the “Charging Helps Agencies Realize General Efficiencies” Act or “CHARGE” Act. This statute requires the U.S. General Services Administration (GSA) to issue guidance on the use of charge cards to pay for charging federally-owned or managed Electric motor Vehicles (EVs) used for official Government business at commercial charging stations. Additionally, this bulletin provides guidance on the use of GSA SmartPay payment solutions for payment of rental car EV charging related to travel on official business. Note that unless otherwise indicated, the mention of specific commercial charging network names in this bulletin is for information purposes only, is not intended as an endorsement of any kind, and does not indicate a contractual relationship with GSA.
```

Comment:
This Smart Bulletin provides operational guidance regarding Public Law 116-160, the “Charging Helps Agencies Realize General Efficiencies” Act, and the “CHARGE” Act. This statute requires the U.S. General Services Administration (GSA) to issue guidance on using charge cards to pay for charging federally owned or managed Electric motor Vehicles (EVs) used for official Government business at commercial charging stations. Additionally, this bulletin guides the use of GSA SmartPay payment solutions for payment of rental car EV charging related to travel on official business. Note that unless otherwise indicated, the mention of specific commercial charging network names in this bulletin is for information purposes only, is not intended as an endorsement of any kind, and does not indicate a contractual relationship with GSA.